pandas/tests/io/test_parquet.py

```diff
@@ -123,6 +123,19 @@ def df_full():
     )
 
 
+@pytest.fixture(
+    params=[
+        datetime.datetime.now(datetime.timezone.utc),
+        datetime.datetime.strptime("2019-01-04T16:41:24+0200", "%Y-%m-%dT%H:%M:%S%z"),
+        datetime.datetime.strptime("2019-01-04T16:41:24+0215", "%Y-%m-%dT%H:%M:%S%z"),
+        datetime.datetime.strptime("2019-01-04T16:41:24-0200", "%Y-%m-%dT%H:%M:%S%z"),
+        datetime.datetime.strptime("2019-01-04T16:41:24-0215", "%Y-%m-%dT%H:%M:%S%z"),
+    ]
+)
+def timezone_aware_date_list(request):
```

Comment:
Do these hit all the branches added to maybe_get_tz?

Reply:
Yes, fastparquet hits with the UTC prefix. Pyarrow hits the .now() without the prefix.

Reply:
IDK `-` without UTC prefix is possible or not, that's missing.

Reply:
Figured it out, using `timezone.min` pyarrow writes `-23:59`